public/docs/getting_started/client.md

```diff
@@ -1,20 +1,54 @@
+# Choosing an Ethereum client
+
 There are many Ethereum clients to choose from. We recommend using different clients when developing and deploying.
 
-# When Developing
+## Developing locally
```

Comment:
I don't think we should highlight "locally" here. We want to paint `truffle develop` (and eventually Ganache) as the what you use to develop, full stop. So "When Developing" fits better, because "Developing locally" suggests you should use something else when developing on a blockchain that exists on a different computer.

Reply:
Oh, I think I'm confused about the scope of Truffle Develop then. I thought it was just a local program you ran when developing. Is that not the case? Or am I confusing the application with the network here?

Reply:
It is local to the computer it runs on, in general, but doesn't necessarily have to be. For instance, some people like to start up an Azure instance that exists somewhere else on the cloud to use Truffle.

From a branding perspective, we want to solidify Truffle Develop as what you use for development. Highlighting "locally" suggests that you'd use something different were you not wanting to develop locally, but that's such a rare case and generally not recommended unless needed.